app/assets/javascripts/custom.js

```diff
@@ -43,8 +50,16 @@ $(document).ready(function(){
     var food_id = $(this).data('message');
     var qty = parseFloat($(this).val());
     var price = parseFloat($('#food_sub_total_' + food_id).data('message'));
+    var prep_time = parseFloat($('#food_prep_' + food_id).data('message'));
     var line_total = parseFloat(qty * price).toFixed(2);
+    var line_prep_total = parseFloat(qty * prep_time);
+
+    $('#food_sub_total_' + food_id).text(line_total);
```

Comment:
Identifier 'food_id' is not in camel case.<br>Identifier 'line_total' is not in camel case.